metafix/src/test/java/org/metafacture/metafix/MetafixMethodTest.java

```diff
@@ -4015,12 +4015,48 @@ public void shouldUriEncodePathSegment() {
             ),
             i -> {
                 i.startRecord("1");
-                i.literal("id", "slash/990223521400206441:DE-A96:61 TYD 16(3)#!");
+                i.literal("id", "/DE-A96:% (3)#!");
                 i.endRecord();
             },
             o -> {
                 o.get().startRecord("1");
-                o.get().literal("id", "slash%2F990223521400206441%3ADE%2DA96%3A61%20TYD%2016%283%29%23%21");
+                o.get().literal("id", "%2FDE-A96%3A%25+%283%29%23%21");
+                o.get().endRecord();
+            }
+        );
+    }
+
+    @Test
+    public void shouldUriEncodePathSegmentWithoutPlusForSpace() {
+        MetafixTestHelpers.assertFix(streamReceiver, Arrays.asList(
+                "uri_encode('id', plus_for_space:'false')"
+            ),
+            i -> {
+                i.startRecord("1");
+                i.literal("id", "/DE-A96:% (3)#!");
+                i.endRecord();
+            },
+            o -> {
+                o.get().startRecord("1");
+                o.get().literal("id", "%2FDE-A96%3A%25%20%283%29%23%21");
+                o.get().endRecord();
+            }
+        );
+    }
+
+    @Test
+    public void shouldUriEncodePathSegmentWithoutSafeChars() {
+        MetafixTestHelpers.assertFix(streamReceiver, Arrays.asList(
+                "uri_encode('id', safe_chars:'', plus_for_space:'false')"
```

Comment:
Drop `plus_for_space` option as this is supposed to test `safe_chars`.

Reply:
Did this now, see 9bf530703ceeb8d80283a6855bc3d4c43e6efaab.